cluster/manifests/roles/poweruser-role.yaml

```diff
@@ -165,6 +165,14 @@ rules:
   - deletecollection
   - patch
   - update
+- apiGroups:
+  - discovery.k8s.io
+  resources:
+  - endpointslices
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
poweruser role implicitly depend on readonly, so it's not needed to add read actions to this role.

Reply:
Thanks, I didn't know that.
The PR is updated to contain changes for `readonly` role only.